src/MovieCard/MovieCard.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
+import Rating from '../Rating/Rating.js';
 import './MovieCard.css';
 
-const MovieCard = ({ title, avgRating, photo, releaseDate, history, movieId }) => {
+
+const MovieCard = ({ movie, history, userId }) => {
```

Comment:
I have also made an issue to refactor to using link here instead of manipulating history per project grading spec.